SparkPost/2web2ui
# SA-1874: Add Contact Support action for Blocklist Remediation

### What Changed

- Added support for blocklist recommendation for Contact Support

### How To Test

- Go to signals/blocklist/incidents. May need to use production environment.
- Click on an incident that involves spamhaus.org
- Go to remediation steps panel and click 'Contact Support'.
- Verify that the other incident remediation steps work as well.

### To Do
- [ ] UX/d12y team feedback
